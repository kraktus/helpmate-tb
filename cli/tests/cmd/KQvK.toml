bin.name = "helpmate_tb_cli"
args = "--no-time generate -m \"KQvK\" --tb-dir ../table"