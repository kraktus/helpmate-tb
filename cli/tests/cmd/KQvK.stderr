dhat: Total:     1,986,038 bytes in 836 blocks
dhat: At t-gmax: 327,430 bytes in 68 blocks
dhat: At t-end:  1,651 bytes in 10 blocks
dhat: The data has been saved to dhat-heap.json, and is viewable with dhat/dh_view.html
